statsmodels/tools/tests/test_grouputils.py

```diff
@@ -76,6 +80,8 @@ def test_transform_dataframe(self):
             np.testing.assert_array_equal(transformed_dataframe,
                                           expected.values)
 
+    @pytest.mark.xfail(condition=PY37,
+                       reason='Unexplained conversion to complex on Python 3.7')
```

Comment:
Can you add `strict=True` to this xfail so we get alerted when these magically fix themselves?